vaadin-notification-flow-parent/vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/Notification.java

```diff
@@ -64,6 +64,8 @@ public class Notification extends Component implements HasComponents, HasStyle,
 
     private static final int DEFAULT_DURATION = 5000;
     private static final Position DEFAULT_POSITION = Position.BOTTOM_START;
+    private static final String OPENED_PROPERTY = "opened";
+    private static final String OPENED_CHANGED_PROPERTY = "opened-changed";
```

Comment:
This is not a property but event name, consider using `OPENED_CHANGED_EVENT`:

```suggestion
    private static final String OPENED_CHANGED_EVENT = "opened-changed";
```

Reply:
done